frontend/rust-lib/flowy-database2/src/services/field/type_options/url_type_option/url_type_option.rs

```diff
@@ -61,28 +61,14 @@ impl TypeOptionCellDataSerde for URLTypeOption {
 }
 
 impl CellDataDecoder for URLTypeOption {
-  fn decode_cell(
-    &self,
-    cell: &Cell,
-    decoded_field_type: &FieldType,
-    _field: &Field,
-  ) -> FlowyResult<<Self as TypeOption>::CellData> {
-    if !decoded_field_type.is_url() {
-      return Ok(Default::default());
-    }
-
+  fn decode_cell(&self, cell: &Cell) -> FlowyResult<<Self as TypeOption>::CellData> {
```

Comment:
Seems like we can remove `decode_cell` in favor of `parse_cell` in all type options with your changes.

Reply:
Yeah... I don't know why make the distinction when implementing it much early on. If it's okay, I'll do it in a follow-up PR.